internal/clusterconfig/clusterconfig.go

```diff
@@ -427,3 +452,39 @@ func (r *UpgradeClusterConfigGather) fetchNetworkConfig(ostreeDir string) error
 	r.Log.Info("Done fetching node network files")
 	return nil
 }
+
+func (r *UpgradeClusterConfigGather) getCustomCatalogSources(ctx context.Context) (operatorsv1alpha1.CatalogSourceList, error) {
+	catalogSources := &operatorsv1alpha1.CatalogSourceList{}
+	customCatalogSources := &operatorsv1alpha1.CatalogSourceList{}
+	allNamespaces := client.ListOptions{Namespace: metav1.NamespaceAll}
+	if err := r.Client.List(ctx, catalogSources, &allNamespaces); err != nil {
+		return operatorsv1alpha1.CatalogSourceList{}, fmt.Errorf("failed to list all catalogueSources %w", err)
+	}
+
+	defaultCatalogRecources := []string{"certified-operators", "community-operators", "redhat-marketplace", "redhat-operators"}
```

Comment:
Maybe align with https://github.com/openshift-kni/lifecycle-agent/blob/e3bd855206b7ec61ec93822217fdd92158b23dc8/lca-cli/seedcreator/seedcreator.go#L478-L485

Reply:
Hmm, I think that one is filtering the images, but let me check with @tsorya

Reply:
yep, this regex matches image names

Reply:
So I think for this case, we just want to filter out the default catalog sources and we achieve that by filtering the catalog source name.